HelloApi/HelloApi/App_Start/WebApiConfig.cs

```diff
@@ -11,6 +13,12 @@ public static void Register(HttpConfiguration config)
         {
             // Web API configuration and services
 
+            var statsdConfig = new StatsdConfig() { StatsdServerName = "127.0.0.1", StatsdPort = 8275 };
```

Comment:
Should I try to read this from config or is that overengineering for this?

Reply:
Also todo figure out where to actually send these

Reply:
I think this is fine considering this is just a sample app that we're going to throw away